tests/libvmi/factory.go

```diff
@@ -129,3 +127,7 @@ func NewWindows(opts ...Option) *kvirtv1.VirtualMachineInstance {
 	vmi.Spec.Domain.Firmware = &kvirtv1.Firmware{UUID: WindowsFirmware}
 	return vmi
 }
+
+func WithAlpineResourceMemory() Option {
```

Comment:
I do not think it is a fit in the factory file.

Reply:
I am not sure where to put this. I do not think this fits the `memory.go` but I'll put it there if you tell me otherwise.

Reply:
It is because it is a "freak" option :).
When it is under a factory, internal, it is part of its logic.

But when it stands alone, it is odd. The caller already decided the need is for a specific VM OS, so she either needs to use the relevant factory and override the rest or create the VM from scratch and re-express the memory logic there.